content/en/docs/tasks/tools/included/kubectl-convert-overview.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "kubectl-convert overview"
+description: "What is kubectl-convert plugin?"
+headless: true
+---
+
+kubectl-convert is a kubectl plugin which allows to convert manifests between different api 
```

Comment:
Let's focus on explaining `kubectl convert`. The binary name `kubectl-convert` is not the important detail here.

Reply:
Something like this would work?
```markdown
A plugin for Kubernetes command-line tool `kubectl`, which allows you to convert manifests between different api 
versions. This can be particularly helpful to migrate manifests to a non-deprecated api version with newer kubernetes release.
For more info, visit [migrate to non deprecated apis](/docs/reference/using-api/deprecation-guide/#migrate-to-non-deprecated-apis)
```

Reply:
Broadly yes. We usually write API in upper case.